routes/itinerary.routes.js

```diff
@@ -0,0 +1,70 @@
+const router = require("express").Router();
+const mongoose = require('mongoose');
+
+const Itinerary = require('../models/Itinerary.model');
+// const Activity = require('../models/Activity.model');
+
+// GET /api/itineraries -  Retrieves all of the itineraries
+router.get('/itineraries', (req, res) => {
+    Itinerary.find()
+    //   .populate('activities')
```

Comment:
had to comment this since it was throwing an error > will figure out what's the problem